dash/development/_py_prop_typing.py

```diff
@@ -0,0 +1,77 @@
+def generate_any(_):
+    return "typing.Any"
+
+
+def generate_shape(t):
```

Comment:
Also, not sure what this function is doing differently than `generate_object_of`, but maybe this is a nested object.

Reply:
`ObjectOf` has a single type while shape can have more than one type for it's keys. This is also more of a placeholder, the final implementation should generate a `TypedDict` or a pydantic model.